sources/packages/web/src/components/common/InstitutionUserSummary.vue

```diff
@@ -92,10 +91,10 @@
             <span class="text-decoration-underline">Edit</span>
           </v-btn>
           <v-btn
-            v-if="canDisableUser(slotProps.data.userName)"
+            :disabled="slotProps.data.disableRemove"
             @click="updateUserStatus(slotProps.data)"
             variant="text"
-            color="primary"
+            :color="slotProps.data.disableRemove ? 'gray' : 'primary'"
```

Comment:
Why are we editing the style of disabled behavior? Is it to match figma? Generally we do not edit the looks of button at it's disabled status. Correct me If I am missing something.

Reply:
Yes, it is to match Figma. Either it is a Vuetify beta issue or the out-of-box behavior, but for these button types the 'disable' does not cause much UI change and does not match Figma.
![image](https://user-images.githubusercontent.com/61259237/179604408-1a95f257-4fc4-4f4b-9267-5661e926db60.png)
As you can see this button demands a bit more of the out-of-box to look a bit better. I am not sure if the same look will be used all around the application but if the out-of-box will not be easy to achieve we can create a component to reuse it.

Reply:
Ah.. I get it.